modules/installation-gcp-user-infra-rhcos.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-infra-rhcos_{context}"]
+= Creating the {op-system} cluster image for the GCP infrastructure
+
+You must use a valid {op-system-first} image for Google Cloud Platform (GCP) for
+your {product-title} nodes.
+
+.Procedure
+
+. Locate the RHCOS image source:
++
+----
+$ export IMAGE_SOURCE=`curl \
+    https://raw.githubusercontent.com/openshift/installer/master/data/data/rhcos.json \
```

Comment:
We don't want this floating-with-master GitHub link in official docs.  Folks should be using something under https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/ or in an an access product page.  For example:

```console
$ git grep 'openshift-v4/dependencies' origin/master
origin/master:modules/installation-osp-creating-image.adoc:. Download the latest {op-system} image from the https://access.redhat.com/downloads/content/290[Product Downloads] page of the Red Hat customer portal or the https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.2/[{op-system} image mirror page].
origin/master:modules/installation-user-infra-machines-iso.adoc:link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.2/[{op-system} image mirror]
origin/master:modules/installation-user-infra-machines-pxe.adoc:link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.2/[{op-system} image mirror]
origin/master:modules/installation-vsphere-machines.adoc:link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.2/[{op-system} image mirror]
```

Reply:
Got this looking like the other similar modules for other provider types:

http://file.rdu.redhat.com/~adellape/100419/gcp_upi/installing/installing_gcp_user_infra/installing-gcp-user-infra.html#installation-gcp-user-infra-rhcos_installing-gcp-user-infra

Probably room to follow-up on this to generalize the module further.